lib/Graphs/PAG.cpp

```diff
@@ -62,6 +105,7 @@ const std::string ObjPN::toString() const {
     std::string str;
     raw_string_ostream rawstr(str);
     rawstr << "ObjPN ID: " << getId();
+    rawstr << "\n";
```

Comment:
Please remove this new line.

Reply:
@yuleisui, I have removed the newline in ObjPN::toString and pushed.